### PR TITLE
Extended bzz schemes part with examples

### PR DESCRIPTION
**Pull request by gbalint**
I added plenty of curl command examples to make the section less abstract and easier to grasp. bzz-resource part is still to be extended/

Closes https://github.com/ethersphere/swarm-guide/issues/76
(because encryption is already assessed in https://github.com/ethersphere/swarm-guide/pull/82)

### CODE REVIEW

File: contents/urlschemes.rst [hunk 1]
```diff
@@ -19,8 +19,54 @@ Example:
...
+    Swarm is awesome%
+
+If the manifest does not contain an file at ``readme.md`` itself, but it does contain multiple entries to which the URL could be resolved, e.g. in the example above, the manifest has entries for ``readme.md.1`` and ``readme.md.2``, the API returns an HTTP response "300 Multiple Choices", indicating that the request could not be unambiguously resolved. A list of available entries is returned via HTTP or JSON.
 
+.. code-block:: none
+
+    $ ls
+    readme.md.1 readme.md.2
+    $ swarm --recursive up .
+    679bde3ccb6fb911db96a0ea1586c04899c6c0cc6d3426e9ee361137b270a463
```
Comment (homotopycolimit) [inline contents/urlschemes.rst]: are we going to upload all of these examples to production once we're live?

Comment (gbalint) [reply contents/urlschemes.rst]: I don't think it is necessary, but we can if it is needed

Comment (homotopycolimit) [reply contents/urlschemes.rst]: might be cool :)

File: contents/urlschemes.rst [hunk 2]
```diff
@@ -56,18 +139,43 @@ bzz-list
...
     GET http://localhost:8500/bzz-hash:/theswarm.eth/
 
-
 Swarm accepts GET requests for bzz-hash url scheme and responds with the hash value of the raw content, the same content returned by requests with bzz-raw scheme. Hash of the manifest is also the hash stored in ENS so bzz-hash can be used for ENS domain resolution.
 
 Response content type is *text/plain*.
 
+.. code-block:: none
+
+    $ curl http://localhost:8500/bzz-hash:/theswarm.test/
```
Comment (homotopycolimit) [inline contents/urlschemes.rst]: let's use .eth instead of .test in the main docs.

File: contents/urlschemes.rst [hunk 3]
```diff
@@ -78,7 +186,19 @@ bzz-immutable
 
 The same as the generic scheme but there is no ENS domain resolution, the domain part of the path needs to be a valid hash. This is also a read-only scheme but explicit in its integrity protection. A particular bzz-immutable url will always necessarily address the exact same fixed immutable content.
 
+.. code-block:: none
 
+    $ curl http://localhost:8500/bzz-immutable:/679bde3ccb6fb911db96a0ea1586c04899c6c0cc6d3426e9ee361137b270a463/readme.md.1
+    ## Hello Swarm!
+
+    Swarm is awesome%
+    $ curl -H "Accept:application/json" http://localhost:8500/bzz-immutable:/theswarm.test/ | jq .
```
Comment (homotopycolimit) [inline contents/urlschemes.rst]: .eth instead of .test

File: contents/urlschemes.rst [hunk 4]
```diff
@@ -78,7 +186,19 @@ bzz-immutable
...
+    {
+        "Msg": "cannot resolve theswarm.test: immutable address not a content hash: \"theswarm.test\"",
```
Comment (homotopycolimit) [inline contents/urlschemes.rst]: .eth

**Pull request merged**